osgp/protocol-adapter-iec60870/README.md

```diff
@@ -10,3 +10,64 @@ Grid eXchange Fabric detailed documentation:
+  VALUES
+    (current_timestamp, current_timestamp, 0, 'LMD_1', 'LMD', true,
+        (SELECT id FROM protocol_info WHERE protocol = '60870-5-104' AND protocol_version = '1.0'),
+        '127.0.0.1', false, (SELECT id FROM device WHERE device_identification = 'GATEWAY_1'),
+        (SELECT id FROM device_model WHERE model_code = 'Test' AND manufacturer_id = (SELECT id FROM manufacturer WHERE code = 'Test')),
+        'IN_USE'),
+    (current_timestamp, current_timestamp, 0, 'LMD_2', 'LMD', true,
+        (SELECT id FROM protocol_info WHERE protocol = '60870-5-104' AND protocol_version = '1.0'),
+        '127.0.0.1', false, (SELECT id FROM device WHERE device_identification = 'GATEWAY_1'),
+        (SELECT id FROM device_model WHERE model_code = 'Test' AND manufacturer_id = (SELECT id FROM manufacturer WHERE code = 'Test')),
+        'IN_USE');
+
+INSERT INTO rtu_device VALUES ((SELECT id FROM device WHERE device_identification = 'GATEWAY_1'), null, 2);
+
+INSERT INTO light_measurement_device
+    (id, description, code, color, digital_input)
+  VALUES
+    ((SELECT id FROM device WHERE device_identification = 'LMD_1'), 'Test LMD 1', 'CODE-1', '#c9eec9', 42),
+    ((SELECT id FROM device WHERE device_identification = 'LMD_2'), 'Test LMD 2', 'CODE-2', '#c9eec9', 78);
+
+INSERT INTO device_authorization
+    (creation_time, modification_time, version, function_group, device, organisation)
+  VALUES
+    (current_timestamp, current_timestamp, 0, 0, (SELECT id FROM device WHERE device_identification = 'GATEWAY_1'), (SELECT id FROM organisation WHERE organisation_identification = 'test-org')),
+    (current_timestamp, current_timestamp, 0, 0, (SELECT id FROM device WHERE device_identification = 'LMD_1'), (SELECT id FROM organisation WHERE organisation_identification = 'test-org')),
+    (current_timestamp, current_timestamp, 0, 0, (SELECT id FROM device WHERE device_identification = 'LMD_2'), (SELECT id FROM organisation WHERE organisation_identification = 'test-org')), 
+    (current_timestamp, current_timestamp, 0, 0, (SELECT id FROM device WHERE device_identification = 'GATEWAY_1'), (SELECT id FROM organisation WHERE organisation_identification = 'LianderNetManagement')),
+    (current_timestamp, current_timestamp, 0, 0, (SELECT id FROM device WHERE device_identification = 'LMD_1'), (SELECT id FROM organisation WHERE organisation_identification = 'LianderNetManagement')),
+    (current_timestamp, current_timestamp, 0, 0, (SELECT id FROM device WHERE device_identification = 'LMD_2'), (SELECT id FROM organisation WHERE organisation_identification = 'LianderNetManagement'));
+
+
+
+-- osgp_adapter_protocol_iec60870
+
+INSERT INTO iec60870_device
+    (creation_time, modification_time, version, device_identification, common_address, port, device_type, gateway_device_identification, information_object_address)
+  VALUES
+    (current_timestamp, current_timestamp, 0, 'GATEWAY_1', 0, 2404, 'LIGHT_MEASUREMENT_GATEWAY', null, null),
```

Comment:
I was wondering if common_address should be 0 or a non-0 value. (Not sure, will have to look it up myself...)